### PR TITLE
Sublime Merge

### PR DESCRIPTION
**Pull request by ptomato**

### CODE REVIEW

File: com.sublimemerge.App.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          - ar x sublime_merge.deb
+          - rm -f sublime_merge.deb
+          - tar xf data.tar.xz
+          - rm -f control.tar.xz data.tar.xz debian-binary
+          - mv usr/* .
+          - rmdir usr
+          - mkdir -p export/share/applications
+          - sed s/Icon=sublime-merge/Icon=com.sublimemerge.App/ share/applications/sublime_merge.desktop > export/share/applications/com.sublimemerge.App.desktop
+          - sed -i 's:/opt/sublime_merge/sublime_merge:sublime_merge:' export/share/applications/com.sublimemerge.App.desktop
+          - mkdir -p export/share/icons/hicolor/16x16/apps
```
Comment (bilelmoussaoui) [inline com.sublimemerge.App.yaml]: You could use install for those and remove the mkdir commands.

File: com.sublimemerge.App.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        dest-filename: apply_extra
+        commands:
+          - ar x sublime_merge.deb
+          - rm -f sublime_merge.deb
+          - tar xf data.tar.xz
+          - rm -f control.tar.xz data.tar.xz debian-binary
+          - mv usr/* .
+          - rmdir usr
+          - mkdir -p export/share/applications
+          - sed s/Icon=sublime-merge/Icon=com.sublimemerge.App/ share/applications/sublime_merge.desktop > export/share/applications/com.sublimemerge.App.desktop
```
Comment (bilelmoussaoui) [inline com.sublimemerge.App.yaml]: You could use desktop-file-edit here

File: com.sublimemerge.App.appdata.xml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+          <li>Windows: Fixed Clone dialog not working correctly</li>
+          <li>Windows: Updated Git Credential Manager to 1.18.1</li>
+          <li>Linux: GTK3 + Wayland compatibility fixes</li>
+          <li>Linux: Fixed GTK3 double and triple click handling</li>
+          <li>Linux: Fixed a crash when using GTK_IM_MODULE=xim</li>
+        </ul>
+      </description>
+    </release>
+  </releases>
+  <content_rating type="oars-1.1">
```
Comment (bilelmoussaoui) [inline com.sublimemerge.App.appdata.xml]: You could use the short version for OARS

File: com.sublimemerge.App.appdata.xml [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+      <image type="source">https://raw.githubusercontent.com/flathub/com.sublimemerge.App/master/screenshot-commit.png</image>
+    </screenshot>
+    <screenshot>
+      <image type="source">https://raw.githubusercontent.com/flathub/com.sublimemerge.App/master/screenshot-search.png</image>
+    </screenshot>
+    <screenshot>
+      <image type="source">https://raw.githubusercontent.com/flathub/com.sublimemerge.App/master/screenshot-conflict.png</image>
+    </screenshot>
+  </screenshots>
+  <categories>
```
Comment (bilelmoussaoui) [inline com.sublimemerge.App.appdata.xml]: Should be set on desktop file only.

**Comment by ptomato** [comment]
Thanks for the review! Here is an updated commit.

**Comment by nedrichards** [comment]
bot, build com.sublimemerge.App

**Comment by bilelmoussaoui** [comment]
Looks pretty good. Maybe just ditch the windows/macOS release description.

**Comment by ptomato** [comment]
Thanks, I updated the release description.

**Comment by nedrichards** [comment]
testbot, build com.sublimemerge.App

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.sublimemerge.App

**Comment by fnkr** [comment]
Hi,

according to the [the Flatpak documentation](http://docs.flatpak.org/en/latest/using-flatpak.html#identifiers) the format for app IDs is `com.company.App`. I assumed that `company.com` is the domain of the vendor and `App` a placeholder for the application name. If this is true, shouldn't the app ID for Sublime Merge then be `com.sublimehq.SublimeMerge`, and for Sublime Text `com.sublimehq.SublimeText` as they are both made by the company Sublime HQ Pty Ltd ([sublimehq.com](https://www.sublimehq.com/))?

The reason I'm asking is that I'm currently Flatpaking [Enpass](https://www.enpass.io/) and I'm not sure whether I should use `io.enpass.App` or `io.sinew.Enpass` as app ID. The vendor is [sinew.io](https://www.sinew.io/), the product website is [enpass.io](https://www.enpass.io/). (I'm not affiliated with that company.)

**Comment by bilelmoussaoui** [comment]
@fnkr You should ask for better info on the #flatpak at freenode. The best answer to this is to contact upstream about the right app-id to use.

**Pull request closed without merging**